test/e2e/performanceprofile/functests/0_config/config.go

```diff
@@ -158,9 +164,18 @@ func externalPerformanceProfile(performanceManifest string) (*performancev2.Perf
 	return profile, nil
 }
 
-func testProfile() *performancev2.PerformanceProfile {
-	reserved := performancev2.CPUSet("0")
-	isolated := performancev2.CPUSet("1-3")
+func testProfile() (*performancev2.PerformanceProfile, error) {
+	//Get cpus capacity on a node; typically it is same for all nodes
+	workerNodes, err := nodes.GetByLabels(testutils.NodeSelectorLabels)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
please either use `Expect` in the code (but please use `ExpectWithOffset` OR return error and let the caller handle - but let's not do both